cirq/ops/fsim_gate.py

```diff
@@ -53,14 +53,22 @@ class FSimGate(gate_features.TwoQubitGate,
         a = cos(theta)
         b = -i·sin(theta)
         c = exp(-i·phi)
+
+    Note the difference in sign conventions between FSimGate and the
+    ISWAP and CZPowGate:
+
+        FSimGate(θ, φ) = ISWAP**(-2θ/π) CZPowGate(exponent=-φ/π)
     """
 
     def __init__(self, theta: float, phi: float):
         """
         Args:
-            theta: The strength of the XX+YY interaction, in radians.
-            phi: CPhase angle in radians. Determines how much the |11⟩ state is
-                phased.
+            theta: Swap angle on the odd 1s subspace, in radians. Determined by
```

Comment:
nit: "odd 1s subspace" is a bit odd.  Suggest just saying "Swap angle on the span({|01>, |10>}) subspace".

Reply:
It was odd indeed. I've evened it out.